test/integration/lib/philipsHue_server.js

```diff
@@ -73,80 +75,20 @@ var philipsHue_server = (function() {
     return light_status[id].state.on;
   }
 
-  function setup(port) {
-
-    var foxboxId;
-
-    _server.use(function (req, res, next) { 
-        // foxbox does not have the json header, so need to 
-        // inject it here so the message body can be properly captured
-        req.headers['content-type'] = 'application/json';
-        last_received_cmd = req.method + ':' + req.originalUrl;
-        next();
-      });
-
-    _server.get('/', function (req, res) {
-      res.status(200).sendFile(
-        path.join(__dirname,'/html/philips_initial.html'));  
-    });
-
-    _server.get('/api/:foxbox/', function (req, res) {   
-         // foxbox id is created within foxbox.  Need to remember this
-         if (foxboxId === undefined) {
-          foxboxId = req.params.foxbox;
-          res.sendStatus(200); 
-        }
-        else {
-          res.status(200).json(api_resp);  
-        }     
-      });
-
-    _server.get('/api/:foxboxId/lights', function (req, res) {
-      if (req.params.foxboxId === foxboxId){ 
-        res.status(200).json(light_status); 
-      }
-      else {
-        res.status(404); 
-        throw 'foxboxId mismatch: got ' + req.params.foxboxId; 
-      }
-    });
+  function pressButton() {
```

Comment:
sure, I'll put the comment.  it mimics the button on the philips hub, after the button press, it assigns the username to the requester.